src/Chunk.h

```diff
@@ -392,18 +392,37 @@ class cChunk :
 	{
 		return m_ChunkData.GetMeta(a_RelX, a_RelY, a_RelZ);
 	}
-	inline void       SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
+	inline void  SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
```

Comment:
> -Or, Just add the two extra parameters to the regular SetMeta and make them default to true, this eliminates the need for metaQuiet-

Right, I was thinking of that when I made my original comment. It may be the better route in this case.